openlr/openlr_match_quality/openlr_assessment_tool/mainwindow.cpp

```diff
@@ -19,15 +26,21 @@
 
 #include <cerrno>
 #include <cstring>
+#include <vector>
 
 namespace
 {
 class TrafficDrawerDelegate : public TrafficDrawerDelegateBase
 {
+  static constexpr char const * kEncodedLineId = "encodedPath";
```

Comment:
How about to replace this by:
```c++
static constexpr char const kEncodedLineId[] = "...";
```
It's more flexible because allows constant-time sizeof.